src/core/Refiner.ml

```diff
@@ -580,6 +558,54 @@ struct
     let+ tp, fam = quantifier tac_base tac_fam univ in
     S.CodeSg (tp, fam)
 
+  let quote_code_sign_hooks (sign : (Ident.user * D.con) list) 
```

Comment:
I don't love this indentation 👑 😆 Is there a way to writ this where `(sign...` starts on a new line?